viewing_party/party.py

```diff
@@ -1,23 +1,146 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        movie_dict = {"title": title, "genre": genre, "rating": rating}
+        return movie_dict
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    '''

```

Comment:
Incredibly minor style nitpick: In some cases you're using single quotes for your docstrings and in other cases you're using double quotes. Python does allow both, but pick one and stick with it to make your code consistent. I would even suggest that whichever you end up using for regular strings be the same one you use for docstrings.

Reply:
I didn't notice you mixing it up for other strings, though!